pkg/controller/infinispan/util/cluster.go

```diff
@@ -250,3 +351,12 @@ func ClusterStatusHandler(scheme corev1.URIScheme) corev1.Handler {
 			Port:   intstr.FromInt(11222)},
 	}
 }
+
+// GetLabelsForInfinispan returns the labels that must me applied to the resource
+func GetLabelsForInfinispan(name, resourceType string) map[string]string {
```

Comment:
@rigazilla are you agree that we will move GetLabelsForInfinispan outside this package as it's not only related to cluster utils?

Reply:
sounds good, let me find a place in common

Reply:
yes, sure

Reply:
uh, we already have the right method in common. removed this